src/HttpServer.ts

```diff
@@ -113,11 +136,56 @@ export class HttpServer {
         }
     }
 
-    statusHandler(req: any, res: any): void {
+    /**
+     * @openapi
+     * /status:
+     *   get:
+     *     summary: It returns a 204 response with an empty body.
+     *     description: It may be used just to check if the server is running.
+     *     responses:
+     *       '204':
+     *         description: No Content.
+     */
+    statusHandler(_: Request, res: Response): void {
         // TODO: check components and return proper status code
         res.status(204).end();
     }
 
+    /**
+     * @openapi
+     * /relay:
+     *   post:
+     *     summary: It relay transactions.
```

Comment:
wdyt of pointing to the rif relay doc here? (devportal)

just to provide some context on what this means. 

idk whether it's bad etiquette to use links in these types of things

Reply:
I don't like this short summary either. but adding a link there doesn't look good, because the summary field doesn't support the markdown, so I added the link in the description